src/ui/actions_main_area.ui

```diff
@@ -224,125 +299,205 @@
+         </item>
+         <item row="4" column="0">
+          <widget class="QLabel" name="label_action_command_up">
+           <property name="text">
+            <string>Command (up):</string>
            </property>
-           <layout class="QHBoxLayout" name="horizontalLayout_4">
-            <property name="spacing">
-             <number>0</number>
-            </property>
-            <property name="margin">
-             <number>0</number>
-            </property>
-           </layout>
           </widget>
          </item>
+         <item row="4" column="1" colspan="5">
+          <widget class="QLineEdit" name="lineEdit_action_command_up">
+           <property name="enabled">
+            <bool>true</bool>
+           </property>
+           <property name="minimumSize">
+            <size>
+             <width>0</width>
+             <height>0</height>
+            </size>
+           </property>
+           <property name="toolTip">
+            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If this is a button that has been set as a &lt;i&gt;push-down button&lt;/i&gt; then what is entered here is sent on the &lt;b&gt;second&lt;/b&gt; press of the button, i.e. going from the &lt;b&gt;on to off&lt;/b&gt; state - in this case the other entry box is only sent every other click as the button goes from the &lt;b&gt;off to on&lt;/b&gt; state.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+           </property>
+          </widget>
+         </item>
+         <item row="5" column="1">
+          <spacer name="verticalSpacer_buttonProperties">
+           <property name="orientation">
+            <enum>Qt::Vertical</enum>
+           </property>
+           <property name="sizeHint" stdset="0">
+            <size>
+             <width>20</width>
+             <height>0</height>
+            </size>
+           </property>
+          </spacer>
+         </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
-    <widget class="QWidget" name="widget_2" native="true">
+    <widget class="QWidget" name="widget_bottom" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
-     <layout class="QHBoxLayout" name="horizontalLayout_3">
+     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <property name="leftMargin">
-       <number>6</number>
-      </property>
-      <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
-       <number>6</number>
-      </property>
-      <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
-       <widget class="QGroupBox" name="groupBox">
+       <widget class="QLabel" name="label_css">
+        <property name="text">
+         <string>CSS:</string>
```

Comment:
The `Qwidget::backgroundVisible` being `true` is intended to show the extent of the content - note that the showSpacesAndTabs / showLineFeedsAndParagraphPilcrow options do not apply to that widget - if you really do not like it I guess we can revert to a plain background of it being false but I like it as it shows up redundant blank lines.

I bet that space is a side-effect of sticking in a spacer to force the remaining stuff to the top of the widget if the space is not all used - when just the "Name:" or showError()/showWarning()/showInformation() wigets are shown for the root icon or a module/package master folder I wanted to avoid them being centred in the right side widget.  Will have a closer look at the nesting of the widget on the right side (there is a splitter somewhere in the mess there...